gem/constants.py

```diff
@@ -0,0 +1,3 @@
+MALE = "m"
+FEMALE = "f"
+GENDERS = {(MALE, "male"), (FEMALE, "female")}
```

Comment:
At minimum, we probably want at least `other` and `prefer not to say` choices as well.